src/components/YandexForm/__stories__/data.json

```diff
@@ -1,7 +1,8 @@
 {
   "default": {
     "content": {
-      "id": "10034371.a94dac9d3bd19f65c9616db0297348ad8d063db0"
+      "id": "61a4e639d4d24e0dbba36f5c",
+      "customFormSection": "cloud"
```

Comment:
is this form somehow related to Yandex Cloud?

Reply:
no, i created it as a regular user without any specific text from Yandex Cloud